### ISSUE TITLE
Hcaptcha

### ISSUE THREAD

**Issue by Brando39** [ad-m/python-anticaptcha#53]
Hi! Today I've encountered on this website (https://www.43einhalb.com/) a different kind of captcha, called hCaptcha, which is very similar to the reCaptcha. Anticaptch already support this kind of captchas.
My question is, will you upgrade the captcha solvers to support the API of that solve them? 
If you won't, how I have to approach to the code to solve it myself? I ask because I'm not sure if I can do it myself or not

Thanks in advance!

**Comment by ad-m** [comment]
If Anticaptcha support it I will release new version of library in next weekend.

**Comment by Brando39** [comment]
Yes they support it!

https://anti-captcha.com/apidoc/hcaptcha

Thanks for the quick response! :)

**Comment by ad-m** [comment]
Could you verify PR #54? 

You can install it using following command:

```
pip install git+https://github.com/ad-m/python-anticaptcha.git@hcaptcha
```

PR contain appropriate documentation (example).

**Comment by Brando39** [comment]
Thanks, it works!

**Comment by VeNoMouS** [comment]
Is this going to merged into the main branch and if so can we get a version increase?

**Comment by ad-m** [comment]
Today in few hours.

**Comment by VeNoMouS** [comment]
cheers @ad-m

**Comment by ad-m** [comment]
@VeNoMouS , sorry for delay. I merged #54, but in #55 on some extra tests. To make tests reliable I have run it very many time and in take time. I hope today merge #55 and release new version.

**Comment by VeNoMouS** [comment]
All good, thanks, I've got users for https://pypi.org/project/cloudscraper asking for hCaptcha support, are you going to be adding in a proxy hCaptcha task ?

**Comment by ad-m** [comment]
I am going to add `HCaptchaTaskProxyless` and `HCaptchaTask` for support hCaptcha . See `examples/hcaptcha_request.py` ( https://github.com/ad-m/python-anticaptcha/blob/master/examples/hcaptcha_request.py ) for detailed usage example.

**Comment by VeNoMouS** [comment]
btw

https://github.com/ad-m/python-anticaptcha/blob/master/python_anticaptcha/__init__.py#L9-L10

dupe defined

**Comment by ad-m** [comment]
Published in v0.5.0. See https://pypi.org/project/python-anticaptcha/ .
